src/platform/plugins/shared/discover/public/application/main/data_fetching/fetch_all.test.ts

```diff
@@ -39,6 +40,15 @@ jest.mock('./fetch_esql', () => ({
   fetchEsql: jest.fn().mockResolvedValue([]),
 }));
 
+jest.mock('@kbn/esql-utils', () => {
+  const original = jest.requireActual('@kbn/esql-utils');
+  return {
+    ...original,
+    // mock mutateQueryStatsGrouping to return the query as is
+    mutateQueryStatsGrouping: jest.fn((query: AggregateQuery) => query),
+  };
+});
+
```

Comment:
```suggestion
```
Is this still necessary? It looks like we don't use `mutateQueryStatsGrouping` in Discover anymore.